Amazon.Ion.ObjectMapper/IonListSerializer.cs

```diff
@@ -10,20 +10,23 @@ public class IonListSerializer : IonSerializer<System.Collections.IList>
         private readonly Type listType;
         private readonly Type elementType;
         private readonly bool isGenericList;
+        private readonly bool permissiveMode;
```

Comment:
Nit: Can you change to start with `is` since this is a boolean variable?

Reply:
Updated